src/super_gradients/training/datasets/detection_datasets/detection_dataset.py

```diff
@@ -232,9 +232,9 @@ def _sub_class_annotation(self, annotation: dict) -> Union[dict, None]:
         :param annotation: Dict representing the annotation of a specific image
         :return:           Subclassed annotation if non empty after subclassing, otherwise None
         """
-        cls_posx = get_cls_posx_in_target(self.original_target_format)
+        cls_index = _get_cls_index_in_target(target_format=self.original_target_format)
```

Comment:
Lt's nt cmprsd wrds in vrbl nd fnctn nms pls

Reply:
[8f5324f](https://github.com/Deci-AI/super-gradients/pull/1327/commits/8f5324f6ae7f39f1836f5cef7630f5c8cf80c417)